lib/publiccloud/ec2.pm

```diff
@@ -156,6 +157,7 @@ sub upload_img {
           . "--ec2-ami '" . $helper_ami_id . "' "
           . "--type '" . $instance_type . "' "
           . "--user '" . $self->provider_client->username . "' "
+          . "--boot-mode '" . $boot_mode . "' "
```

Comment:
Wouldn't it be better to not set this at all, if the `PUBLIC_CLOUD_EC2_BOOT_MODE` setting is not set?

Reply:
ok, fine by me!